PugSharp/CsServer.cs

```diff
@@ -206,18 +214,66 @@ public void UnpauseMatch()
         ExecuteCommand("mp_unpause_match");
     }
 
+    public async Task InitializeWorkshopMapLookupAsync()
+    {
+        if (_WorkshopMapLookup != null)
+            return; // Already loaded
```

Comment:
One short remark. We usually force the brackets even i there is only one line :D